Dockerfile

```diff
@@ -1,21 +1,35 @@
 # sections ordered from least likely to change to most
 FROM cfmeqe/sel_base:latest
+
+ENV CHROME_DRIVER_VERSION 2.21
```

Comment:
Like this a lot! I presume this sets a default? We should see what the current FF supported version is. Both the chrome driver and selenium bump seemed to work with the framework OK?